src/meta/meta_ckpt.c

```diff
@@ -172,6 +174,30 @@ __wt_meta_checkpoint_last_name(
     return (ret);
 }
 
+/*
+ * __ckpt_parse_time --
+ *     Parse clock time from checkpoint metadata config. This requires special handling because
+ *     times are unsigned values and config parsing treats numeric values as signed.
+ */
+static int
+__ckpt_parse_time(WT_SESSION_IMPL *session, WT_CONFIG_ITEM *a, uint64_t *timep)
+{
+    char timebuf[64];
+
+    WT_UNUSED(session);
```

Comment:
Do we need to pass in a `WT_SESSION_IMPL` arg if it will never be used? Is it for consistency?

Reply:
Yes, I'm just including the `WT_SESSION_IMPL` for consistency. Most WT methods take a session argument, and it seems easier to just include one here instead of forcing developers to treat this function differently. This is, of course, not much overhead with modern IDEs but we do this in a bunch of other places in WT code, so that's what I did here.